### PR TITLE
Niloofar / Fix ts errors on academy/components

### PR DESCRIPTION
**Pull request by niloo-fs**
Changes:

-   ...

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/common/utility.ts [hunk 1]
```diff
@@ -281,7 +281,8 @@ export const redirectToTradingPlatform = () =>
 
 // Function to manually add external js files.
 type TSettings = {
-    src: 'https://static.deriv.com/scripts/cookie.js'
+    src?: 'https://static.deriv.com/scripts/cookie.js'
+    text?: string
```
Comment (iman-fs) [inline src/common/utility.ts:285]: Do we need this?

**Review by iman-fs — APPROVED** [review]

File: src/common/utility.ts (hunk 1 above)
Comment (iman-fs) [inline src/common/utility.ts:285]: do we need this?

Comment (niloo-fs) [reply src/common/utility.ts:285]: I guess yes, because they are passing a text to it in one of the files

**Review by eduard-deriv — APPROVED** [review]

**Comment by eduard-deriv** [comment]
check all files, LGTM! Interesting solutions in some places 👍

**Pull request merged**